_dashboards/quickstart.md

```diff
@@ -1,120 +1,119 @@
 
-1. On the **[Flights] Airline Carrier** panel, choose **OpenSearch-Air**. The dashboard updates automatically.
-1. Choose **Save** to save the customized dashboard.
+The following image shows the view you see once you have completed the preceding steps.
 
-Alternatively, you can apply filters using the dashboard toolbar:
+<img src="{{site.url}}{{site.baseurl}}//images/<insert image>" alt="Screenshot of Dashboard tutorial panel view" width="700">
 
-1. In the dashboard toolbar, choose **Add filter**.
-1. From the **Field**, **Operator**, and **Value** dropdown lists, choose **Carrier**, **is**, and **OpenSearch-Air**, respectively, as shown in the following image.
+## Turning on and off dark mode
 
-    <img src="{{site.url}}{{site.baseurl}}/images/edit-filter.png" alt="Edit field interface" width="400"/>
+Turning on and off dark mode in OpenSearch Dashboards requires administrative access. If you are an administrator, follow these steps:
 
-1. Choose **Save**. The dashboard updates automatically, and the result is the dashboard shown in the following image.
+1. Navigate to **Management** > **Dashboards Management** > **Advanced Settings**.
+2. Scroll down to the **Appearance** section and locate the **Dark mode** option.
+3. Use the toggle switch to turn on or turn off dark mode for all users of your OpenSearch Dashboards instance.
+4. Select the **Save changes** button and then the **Reload** button. Changes take effect immediately, so you will see the updated theme right away, similar to the view shown in the following image.
 
-  <img src="{{site.url}}{{site.baseurl}}/images/interact-filter-dashboard.png" alt="Dashboard view after applying Carrier filter" width="700"/>
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dark-mode.png" alt="Dark mode view" width="700">
 
 ## Next steps
 
-- **Visualize data**. To learn more about data visualizations in OpenSearch Dashboards, see [**Building data visualizations**]({{site.url}}{{site.baseurl}}/dashboards/visualize/viz-index/).
-- **Create dashboards**. To learn more about creating dashboards in OpenSearch Dashboards, see [**Creating dashboards**]({{site.url}}{{site.baseurl}}/dashboards/quickstart-dashboards/).
-- **Explore data**. To learn more about exploring data in OpenSearch Dashboards, see [**Exploring data**]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/). 
\ No newline at end of file
+- **Visualize data.** To learn more about data visualizations in OpenSearch Dashboards, go to [Building data visualizations]({{site.url}}{{site.baseurl}}/dashboards/visualize/viz-index/).
+- **Create dashboards.** To learn more about creating dashboards in OpenSearch Dashboards, go to [Creating dashboards]({{site.url}}{{site.baseurl}}/dashboards/quickstart-dashboards/).
+- **Gain data insights.** To learn more about your data in OpenSearch Dashboards, go to [Exploring data]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/). 
```

Comment:
I would change the name of the linked page to "Analyzing data" and take a look at our word choices across the Dashboards documentation. We seem to do a lot of "exploring," "discovering," and "unlocking," when it would be better to use a word that reflects what the user will actually _do_.

Reply:
> I would change the name of the linked page to "Analyzing data" and take a look at our word choices across the Dashboards documentation. We seem to do a lot of "exploring," "discovering," and "unlocking," when it would be better to use a word that reflects what the user will actually _do_.

@natebower I created an issue based on this comment. https://github.com/opensearch-project/documentation-website/issues/6531